<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>EditPatch</class>
 <widget class="QWidget" name="EditPatch">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>703</width>
    <height>357</height>
   </rect>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <layout class="QHBoxLayout" name="buttonsLayout">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QToolButton" name="previousFile">
         <property name="toolTip">
          <string>Previous file</string>
         </property>
         <property name="icon">
          <iconset theme="arrow-left">
           <normaloff>.</normaloff>.</iconset>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QToolButton" name="previousHunk">
         <property name="toolTip">
          <string>Previous difference</string>
         </property>
         <property name="icon">
          <iconset theme="arrow-up">
           <normaloff>.</normaloff>.</iconset>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QToolButton" name="nextHunk">
         <property name="toolTip">
          <string>Next difference</string>
         </property>
         <property name="icon">
          <iconset theme="arrow-down">
           <normaloff>.</normaloff>.</iconset>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QToolButton" name="nextFile">
         <property name="toolTip">
          <string>Next file</string>
         </property>
         <property name="icon">
          <iconset theme="arrow-right">
           <normaloff>.</normaloff>.</iconset>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>60</width>
           <height>21</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QSpinBox" name="diffContext">
         <property name="toolTip">
          <string>lines of context for the unified diff</string>
         </property>
         <property name="specialValueText">
          <string>max</string>
         </property>
         <property name="suffix">
          <string extracomment="lines of context text"/>
         </property>
         <property name="minimum">
          <number>0</number>
         </property>
         <property name="maximum">
          <number>999</number>
         </property>
         <property name="value">
          <number>3</number>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="updateButton">
         <property name="toolTip">
          <string>Click to update the patch
Right-click to select the number of lines of context the patch should show.</string>
         </property>
         <property name="text">
          <string>Update</string>
         </property>
         <property name="icon">
          <iconset theme="view-refresh">
           <normaloff>.</normaloff>.</iconset>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="testsButton">
         <property name="text">
          <string>Run Tests</string>
         </property>
         <property name="icon">
          <iconset theme="preflight-verifier">
           <normaloff>.</normaloff>.</iconset>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <spacer name="buttonsSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeType">
        <enum>QSizePolicy::Fixed</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_3">
       <item>
        <widget class="QToolButton" name="exportReview">
         <property name="text">
          <string>Export Diff...</string>
         </property>
         <property name="popupMode">
          <enum>QToolButton::InstantPopup</enum>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="cancelReview">
         <property name="text">
          <string>Cancel Review</string>
         </property>
         <property name="icon">
          <iconset theme="dialog-cancel">
           <normaloff>.</normaloff>.</iconset>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QToolButton" name="finishReview">
         <property name="text">
          <string>Finish Review</string>
         </property>
         <property name="toolButtonStyle">
          <enum>Qt::ToolButtonTextBesideIcon</enum>
         </property>
         <property name="autoRaise">
          <bool>false</bool>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="contentLayout">
     <item>
      <widget class="QGroupBox" name="filesGroup">
       <property name="title">
        <string>Changes</string>
       </property>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <layout class="QVBoxLayout" name="verticalLayout">
          <item>
           <widget class="QProgressBar" name="testProgressBar">
            <property name="value">
             <number>0</number>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QTreeView" name="filesList">
            <property name="editTriggers">
             <set>QAbstractItemView::NoEditTriggers</set>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
